test/resources/compliance_history/policy-uninstall-gk.yaml

```diff
@@ -10,32 +10,54 @@ spec:
         apiVersion: policy.open-cluster-management.io/v1
         kind: ConfigurationPolicy
         metadata:
-          name: compliance-api-uninstall-gk
+          name: compliance-api-uninstall-gk-csv
         spec:
           object-templates-raw: |
-            {{- $csvName := (lookup "operators.coreos.com/v1alpha1" "Subscription" "openshift-operators" "gatekeeper-operator-product").status.installedCSV -}}
+
+            {{ range $ns := (lookup "v1" "Namespace" "" "").items }}
+            {{ range $csv := (lookup "operators.coreos.com/v1alpha1" "ClusterServiceVersion"  "" "").items }}
+            {{ $csvName := $csv.metadata.name }}
+            {{ if hasPrefix "gatekeeper-operator-product." $csvName }}
```

Comment:
Nice :wink:

Reply:
Thanks! 😄 

Though our standup today made me think: Should the Subscription be deleted first or does that really matter (and whether that might be contributing to the processing time)? Seems I could swap the dependencies here so that the Subscription is deleted followed by the CSV if OLM really is helping clean up the CSVs...

Reply:
Oh I missed that, yes, you need to delete the Subscription first or else OLM will recreate the CSVs.

Reply:
Oops. That's probably why it takes 5 minutes 🤦

Reply:
Actually, even swapping them seems to take a long time--I've opened a PR to swap it. I now suspect it may be the dependencies that are taking too long, but I've opened an issue.